tasks/main.yml

```diff
@@ -9,11 +9,24 @@
   template:
     src: exim.conf.j2
     dest: /etc/exim4/exim4.conf
-  notify: Enable and reload exim4
+  notify: Reload exim4
+
+- name: Create exim_ansible_role directory
+  file:
+    path: /etc/exim_ansible_role
+    state: directory
+
+- name: Place extra files
+  copy: "{{ extra_file | combine({ 'dest': '/etc/exim_ansible_role/' + extra_file.dest }) }}"
```

Comment:
Why the separate dir? The files could be placed in `/etc/exim4` which already exists and holds files necessary for exim

Reply:
There might be circumstances where a file deployed by this role should be read by multiple softwares. For instance, when using a mailing list software that supports ARC, then that software needs access to the DKIM private key in order to sign ARC seals.

Reply:
However, if you still think that this should still be in `/etc/exim4`, then I can make another PR that realizes you wish.